packages/sirius-web/frontend/sirius-web-application/src/router/Router.tsx

```diff
@@ -29,10 +30,11 @@ export const Router = () => {
       <Route exact path="/projects" component={withErrorBoundary(ProjectBrowser)} />
       <Route exact path="/projects/:projectId/edit/:representationId?" component={withErrorBoundary(EditProjectView)} />
       <Route exact path="/projects/:projectId/settings" component={withErrorBoundary(ProjectSettingsView)} />
+      <Route exact path="/errors/:code" component={ErrorView} />
       {routes.map((props, index) => (
         <Route key={index} {...props} />
       ))}
-      <Redirect to="/projects" />
+      <Redirect to="/errors/404" />
```

Comment:
Now that the default redirection is to the errors page, the 'back to homepage' button at top left no longer redirects to the projects page.
``` 
 <Tooltip title="Back to the homepage">
              <Link component={RouterLink} to="/" className={classes.link} color="inherit">

Reply:
Done